VideoLocker/src/main/java/org/edx/mobile/view/WebViewFindCoursesActivity.java

```diff
@@ -69,9 +76,54 @@ public boolean onQueryTextChange(String newText) {
                 return false;
             }
         });
+        searchView.setOnQueryTextFocusChangeListener(new View.OnFocusChangeListener() {
+            @Override
+            public void onFocusChange(View view, boolean queryTextFocused) {
+                if (!queryTextFocused) {
+                    searchActive = false;
+                    searchItem.collapseActionView();
```

Comment:
Looks like the only reasonable way to collapse just the view is to actually invoke the `onActionViewCollapsed()` directly. This also clears the focus from the view, thus automatically causing it's keyboard to be dismissed. So I guess we should replace the `dismissKeyboard()` method with this call instead.